qtpy/QtCore.py

```diff
@@ -8,18 +8,29 @@
 """
 Provides QtCore classes and functions.
 """
-
 from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6, PythonQtError
 
+
 if PYQT6:
+    from PyQt6 import QtCore
     from PyQt6.QtCore import *
     from PyQt6.QtCore import pyqtSignal as Signal
+    from PyQt6.QtCore import pyqtBoundSignal as SignalInstance
+    from PyQt6.QtCore import pyqtSlot as Slot
+    from PyQt5.QtCore import pyqtProperty as Property
```

Comment:
```suggestion
    from PyQt6.QtCore import pyqtProperty as Property
```

Oops, typo? This is what is causing the tests to fail under PyQt6.